apps/omg/test/omg/state/core_test.exs

```diff
@@ -736,6 +736,40 @@ defmodule OMG.State.CoreTest do
     |> fail?(:utxo_not_found)
   end
 
+  @tag fixtures: [:alice, :bob, :carol]
+  test "getting user utxos from utxos_query_result", %{
```

Comment:
1. add more unit-testey cases here (like no utxos that belong to an address within the query result and empty query result).
2. add (or expand existing) test to the persistence tests of state, to make sure that the `utxos_query_result` is in line with the `Core` function